mdl/src/main/cft/mdl.yml

```diff
@@ -36,11 +36,11 @@ Parameters:
     AllowedValues: [true, false]
     Type: String
   HerdVersion:
-    Default: '0.81.0'
+    Default: '0.107.0'
     Description: Herd release version to use.
     Type: String
   HerdUIVersion:
-    Default: '0.41.0'
+    Default: '0.73.0'
```

Comment:
should this be 77

Reply:
I think I checked and 77 was not available on npm registry at that time, I can check again though

Reply:
looks like it is available, let me test it out and update my PR. Thanks